src/hooks/useAccountBalance.ts

```diff
@@ -124,7 +172,9 @@ export const useAccountBalance = ({
     : evmTokenBalance?.result !== undefined && evmTokenDecimals?.result !== undefined
       ? formatUnits(evmTokenBalance?.result, evmTokenDecimals?.result)
       : undefined;
-  const balance = isCosmosChain ? cosmosQuery.data : evmBalance;
+
+  const cosmosBalance = parseFloat(cosmosQuery.data ?? '0') - COSMOS_GAS_RESERVE;
```

Comment:
`cosmosBalance` -> `selectedChainBalance` (I think may be a little more descript). Instead of dealing with negatives for this var, I think we just check for `cosmosQuery.data` and make it `undefined`, then do an `undefined` check for `const balance`.